.github/workflows/publish.yml

```diff
@@ -0,0 +1,32 @@
+name: Publish
+
+on:
+  push:
+    branches:
+      - 'release/*.*.x'
```

Comment:
I was wondering whether we should maintain minor branches, e.g. `1.1.x`.

For major branches it makes sense as breaking changes are introduced.

We can have a think later, for now looks good.

Reply:
Let's discuss about this on Monday, before the 1st release.